e2e_tests/tests/cluster/test_checkpoints.py

```diff
@@ -81,14 +87,15 @@ def run_gc_checkpoints_test(checkpoint_storage: Dict[str, str]) -> None:
             trials = exp.experiment_trials(experiment_id)
             assert len(trials) == 1
 
+            checkpoints = list(filter(lambda w: "checkpoint" in w, trials[0]["workloads"]))
             checkpoints = sorted(
-                (step["checkpoint"] for step in trials[0]["steps"]),
-                key=operator.itemgetter("total_batches"),
+                (workload["checkpoint"] for workload in checkpoints),
+                key=operator.itemgetter("totalBatches"),
             )
             assert len(checkpoints) == 10
             by_state = {}  # type: Dict[str, Set[int]]
             for checkpoint in checkpoints:
-                by_state.setdefault(checkpoint["state"], set()).add(checkpoint["total_batches"])
+                by_state.setdefault(checkpoint["state"], set()).add(checkpoint["totalBatches"])
```

Comment:
question: aren't these coming from our generated code?  why do we access them this way and not by the dot notation?